Changes

```diff
@@ -1,6 +1,7 @@
 Revision history for {{$dist->name}}
 
 {{$NEXT}}
+  - Support dosftp listings with four-digit years
```

Comment:
```suggestion
  - Support dosftp listings with four-digit years (gh#26)
```

Reply:
Just noticed this should fix #3

Reply:
No problem! I should be able to update the PR later today or tomorrow. Thanks!